app/src/main/java/com/resala/mobile/qrregister/ui/navhostactivity/NavHostActivity.kt

```diff
@@ -13,10 +15,13 @@ import org.koin.android.viewmodel.ext.android.viewModel
 class NavHostActivity : BaseActivity<NavHostActivityVm>() {
 
     override val vm: NavHostActivityVm by viewModel()
-    override var layoutId: Int = R.layout.activity_nav_host
+    override var layoutId: Int = R.layout.login_frag
+    override fun onCreate(savedInstanceState: Bundle?) {
+        super.onCreate(savedInstanceState)
+        setContentView(layoutId)
+    }
```

Comment:
no need for these lines if you just want to set content view 
` override var layoutId: Int = R.layout.login_frag` do the job of setting content view